### ISSUE TITLE
Delete function missing

### ISSUE THREAD

**Issue by lorentrogers** [QTodoTxt/QTodoTxt#337]
I'm running 1.6.2 on Arch linux with i3 window manager, and I'm not seeing the delete function for tasks. I remember pressing the delete key and having it ask if I want to remove the task. But I don't see it now. Was this removed?

**Comment by oroulet** [comment]
Look at options. We changed back default, and need to make new t release

**Comment by PKEv** [comment]
@lorentrogers via the settings menu turned back?

**Comment by lorentrogers** [comment]
Aha -- I remember this now. There's definitely a bug. We talked about making this the default back in #304. Looks like there's an initialization issue.

I'm running the git repo master branch (commit 565bb1d) from a fresh install of Arch linux. When I launched the application, the "enable delete" setting in the preferences was checked but the delete option was not being shown. I *un*checked it, then restarted the application and nothing had changed. However, when I *checked* it and restarted again, the delete option appeared in the top menu.

It looks like it's not being initialized during install, and the state is only read once it changes. 

https://github.com/QTodoTxt/QTodoTxt/tree/565bb1d2e34391aa72a0a82b37053d94d4c45229
